test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -257,32 +257,36 @@ std::string RandomCodeBase::generate(int _maxOpNumber, RandomCodeOptions const&
 		eth::Instruction inst = (eth::Instruction) opcode;
 		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+		while (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
 			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
 		{
 			if (_options.useUndefinedOpCodes)
+			{
 				code += toCompactHex(opcode, 1);
+				return "0x" + code;
```

Comment:
Why not `continue` and fill more opcodes?

Reply:
it is continued in a for loop

Reply:
No, when you say `return`, the execution returns from the function.

You should continue in the for-loop.  Do you know `break` in loops?

Reply:
hm your right. but it should exit the while loop and continue the for loop then

Reply:
Check what `break` does in loops.